src/sync_key_gen.rs

```diff
@@ -10,7 +10,89 @@
 //! signatures and encryption. The secret master key is not known by anyone. The protocol succeeds
 //! if up to `threshold` nodes are faulty.
 //!
-//! # How it works
+//! ## Usage example
+//!
+//! ```
```

Comment:
Doc code should be prefixed with ` ```rust,ignore ` or ` ` ```rust,no_run ` (to compile only). Ideally though it's better to let it run it that way it stays up to date.

Reply:
Yes, in this case I wrote the example in a way that it runs and passes. (`` ``` `` is equivalent to `` ```rust ``.)

I hope that doesn't go too much at the expense of helpfulness and readability. An alternative would be to write an example that's more like what you would write in an actual application, instead of simulating several nodes.

Reply:
Oh, ` ``` ` is equivalent now? I guess I missed that change to rustdoc.

Reply:
Yeah nevermind I'm just mixing it up with regular markdown. Ignore me.